react/01-react-hw/src/components/App.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import './App.css';
+import UserForm from './UserForm';
+import Greetings from './Greetings';
+
+class App extends React.Component {
+  state = {
```

Comment:
it's ok for now, but App component should not contain some of the screen-specific logic